pkg/storaged/format-dialog.jsx

```diff
@@ -61,27 +61,28 @@ export function crypto_options_dialog_fields(options, visible, include_store_pas
     var extra_options = unparse_options(split_options);
 
     var fields = [
-        { title: _("Unlock at boot"), tag: "auto" },
-        { title: _("Unlock read only"), tag: "ro" },
-        { title: _("Custom encryption options"), tag: "extra", type: "checkboxWithInput" },
+        { value: { auto: opt_auto }, field: { title: _("Unlock at boot"), tag: "auto" } },
+        { value: { ro: opt_ro }, field: { title: _("Unlock read only"), tag: "ro" } },
+        {
+            value: { extra: extra_options === "" ? false : extra_options },
+            field: { title: _("Custom encryption options"), tag: "extra", type: "checkboxWithInput" }
+        },
     ];
 
     if (include_store_passphrase)
-        fields = [{ title: _("Store passphrase"), tag: "store_passphrase" }].concat(fields);
-
-    return [
-        CheckBoxes("crypto_options", "",
-                   {
-                       visible: visible,
-                       value: {
-                           auto: opt_auto,
-                           ro: opt_ro,
-                           extra: extra_options === "" ? false : extra_options
+        fields = [{ field: { title: _("Store passphrase"), tag: "store_passphrase" } }].concat(fields);
+
+    return fields.map(field => {
```

Comment:
I don't understand this, I am afraid. The old code rturned just one `CheckBoxes` element, the new code returns multiple, one for each actual checkbox input.  Why does this need to be changed?

Reply:
its' the same difference you noticed here. https://github.com/cockpit-project/cockpit/pull/15261#issuecomment-779763836

Checkboxes can either belong to one label a a group, and in that case they should be rendered inside a `FromGroup` component. However, Checkboxes often are not part of one label, how the crypto options are, in this case they should be rendered one by one, each wrapped by a `FormGroup` element.

Reply:
They belong to a group (except "Store passphrase"), but they didn't have a label. What about this:

![image](https://user-images.githubusercontent.com/3228183/108060105-1d41fd00-705f-11eb-960b-bb8837f38550.png)

Reply:
I am happy with this actually! Makes sense!

Reply:
This change is also responsible for at least one test failure.